plopfile.js

```diff
@@ -81,6 +81,11 @@ module.exports = function (plop) {
         path: "packages/paste-core/{{component-type}}/{{kebabCase component-name}}/tsconfig.json",
         templateFile: "tools/plop-templates/tsconfig.hbs",
       },
+      {
+        type: "add",
+        path: "packages/paste-core/{{component-type}}/{{kebabCase component-name}}/CHANGELOG.md",
+        templateFile: "tools/plop-templates/CHANGELOG.hbs",
+      },
```

Comment:
This is a mandatory file that is needed when developing the website section for new components. Good to add it in at gen level to stop getting caught out and manually having to update the components package later.